domain/src/main/java/me/androidbox/domain/authentication/usecase/SaveFullNameUseCase.kt

```diff
@@ -0,0 +1,5 @@
+package me.androidbox.domain.authentication.usecase
+
+fun interface SaveFullNameUseCase {
```

Comment:
Why are you saving it separately?

You could just have 1 method being "SaveUserInformation" or something similar, that just saves all 3 attributes at once

Reply:
@MKiperszmid Updated to have a single method to save the current logged in user